rollup.config.js

```diff
@@ -38,56 +38,57 @@ const outputConf = {
   sourcemap: withSourceMaps,
 };
 
+// eslint-disable-next-line no-console
 console.log('ENV', ENV);
 
-const commonConfig = config => {
-  return {
-    external: [...Object.keys(pkg.dependencies)],
-    plugins: [
-      resolve({
-        preferBuiltins: false,
-      }),
-      alias({
-        '@': resolveDir('./src'),
-        resolve: ['.js', '/index.js'],
-      }),
-      json(),
-      replace({
-        ENV: JSON.stringify(ENV),
-      }),
-      ts(),
-      babel({
-        runtimeHelpers: true,
-        exclude: 'node_modules/**',
-        extensions: ['.js', '.ts'],
-      }),
-      commonjs(),
-      !withSourceMaps && terser(),
-      config.withCopy &&
-        copy({
-          targets: [
-            {
-              src: config.withCopy,
-              dest: './dist',
-            },
-          ],
-        }),
-      postcss({
-        plugins: [
-          url({
-            url: 'inline',
-            encodeType: 'base64',
-            optimizeSvgEncode: true,
-          }),
+const commonConfig = config => ({
+  external: [...Object.keys(pkg.dependencies)],
+  plugins: [
+    resolve({
+      preferBuiltins: false,
+    }),
+    alias({
+      '@': resolveDir('./src'),
+      resolve: ['.js', '/index.js'],
+    }),
+    json(),
+    replace({
+      ENV: JSON.stringify(ENV),
+    }),
+    ts(),
+    babel({
+      runtimeHelpers: true,
```

Comment:
I do not like this because runtimeHelpers makes final bundle bloat. May be remove that and do that only for `umd` builds? I think `esm` or `commonJs` modules do not need that, because they would be processed by build tool on client side.

Reply:
hmm, will try to check this by using different configs

Reply:
i tired to make different builds, but it's not so obvious. 
because we have babel.config for the whole project, and when starting compiling with rollup, bundler is throw error
```
Error: Runtime helpers are not enabled. Either exclude the transform-runtime Babel plugin or pass the `runtimeHelpers: true` option.
```
😕

maybe for the next time, but not for this pr, i think